src/shipit_static_analysis/shipit_static_analysis/revisions.py

```diff
@@ -50,6 +51,10 @@ def analyze_files(self, repo):
             for filename, diff in patch.items()
         }
 
+        # Report nb of files and lines analyzed
+        stats.api.increment('analysis.files', len(self.files))
+        stats.api.increment('analysis.lines', sum(len(line) for line in self.lines.values()))
```

Comment:
Nit: this can be confusing as you would expect `line` to be 1 😄 
Maybe `sum(len(diff) for diff in self.lines.values())`.